### ISSUE TITLE
Please put font size in settings.

### ISSUE THREAD

**Issue by Zibri** [sky-map-team/stardroid#494]
Please put font size in settings.
The font is too smal on higher definition phones.
And for some people need adjustment.
Also: people wearing glasses use glasses for far when they walk around so it's more difficult to see something near.

**Comment by CraftyStarGoddess** [comment]
Yes please put font size options somewhere. My aging eyes cannot see tiny red text in the dark. Thank you!!

**Comment by jaydeetay** [comment]
Same!  What this is blocked on, is basically a rewrite of the rendering
code...so it's not as straightforward as I wish.

**Comment by jaydeetay** [comment]
This has actually been implemented a few versions back.
